vta/apps/tsim_example/python/tsim.py

```diff
@@ -21,15 +21,24 @@
 from sys import platform
 
 def get_ext():
+    """Return shared library extension"""
     return ".dylib" if platform == "darwin" else ".so"
 
 def load_dll(dll):
+    """Load shared library
+
+     Parameters
```

Comment:
Please mind the indentation, leave four spaces before `Parameters` to align with `"""`

Reply:
Fixed